src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -552,6 +650,20 @@ ov::AnyMap get_default_generate_config(const std::shared_ptr<ov::Model>& model,
     return config;
 }
 
+ov::AnyMap get_default_tail_mm_config(const std::optional<NPUDesc>& npudesc) {
+    auto config = get_default_common_config(npudesc);
+    config.erase("NPUW_SLICE_OUT");
+    config.emplace("NPUW_ONLINE_PIPELINE", "NONE");
+    if (npudesc.has_value() && npudesc->arch == "4000") {
+        config.emplace("NPU_DPU_GROUPS", 4);
+    }
```

Comment:
please don't do this here. Probably there's no need to do this at all. Also, `NPU_DPU_GROUPS` were removed recently; this must be `NPU_TILES`. I believe on LNL `4` is the default so we don't need to set it?

Reply:
Removed, thanks!